docs/buildstrategies.md

```diff
@@ -155,4 +159,313 @@ kubectl apply -f samples/buildstrategy/source-to-image/buildstrategy_source-to-i
 [hubcloudfoundry]: https://hub.docker.com/r/cloudfoundry/cnb
 [kaniko]: https://github.com/GoogleContainerTools/kaniko
 [s2i]: https://github.com/openshift/source-to-image
-[buildah]: https://github.com/containers/buildah
\ No newline at end of file
+[buildah]: https://github.com/containers/buildah
+
+## Steps Resource Definition
+
+All strategies steps can include a definition of resources(_limits and requests_) for CPU, memory and disk. For strategies with more than one step, each step(_container_) could require more resources than others. Strategy admins are free to define the values that they consider the best fit for each step. Also multiple strategies of the same type can be _installed_ on the cluster, each with different resources values for the steps.
+
+### Strategies with different resources
+
+If the strategy admins would require to have multiple flavours of the same strategy, where one strategy have more resources that the other. Then, multiple strategies for the same type should be defined on the cluster. In the following example, we use Kaniko as the type:
```

Comment:
`where one strategy have more resources that the other` -> `where one strategy has more resources than the others`

Reply:
is like the joke, when you speak more than one language u are not `bilingual`, you become `byelingual`